akka-stream-tests/src/test/scala/akka/stream/io/DeprecatedTlsSpec.scala

```diff
@@ -0,0 +1,540 @@
+/*
+ * Copyright (C) 2018-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.io
```

Comment:
this is just a copy of old test to keep test coverage of deprecated methods, not much to review